README.md

```diff
@@ -1,59 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Setup Project
 
-> One paragraph statement about the project.
+> This project is about setting the linters, html, css setup.
 
-![screenshot](./app_screenshot.png)
+![Screenshot from 2022-01-08 16-00-44](https://user-images.githubusercontent.com/75738563/148640390-91c48bb9-a520-4cb1-804c-3e7e6d693e62.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: html and css
+- Frameworks: No
+- Technologies used: vs code, github, Firefox
 
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
[Optional]
- This live demo link is broken. Kindly fix this or remove it. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.